ddtrace/contrib/_loguru.py

```diff
@@ -16,7 +16,8 @@
 Patch ``loguru``
 ~~~~~~~~~~~~~~~~~~~
 
-If using :ref:`ddtrace-run<ddtracerun>` then set the environment variable ``DD_LOGS_INJECTION=true``.
+Loguru support is auto-enabled when :ref:`ddtrace-run<ddtracerun>` is used.
```

Comment:
nit: can you add the `and a structured logging format  (ex: JSON) is used.` note here too?